### DISCUSSION TITLE
Rendre une source indisponible avec champs multiples date début/fin

### DISCUSSION THREAD

**Discussion by taib27** [JeromeDevome/GRR#362] [Q&A]
**Votre demande de fonctionnalité est-elle liée à un problème ? Décrivez s'il vous plait.**
Déclarer cette ressource temporairement non disponible. Les réservations sont alors impossibles. La restriction ne s'applique pas aux gestionnaires de la ressource.

**Décrivez la solution que vous souhaitez**
Rendre une source indisponible pour une durée déterminer (date début - Fin  // multiple )
Exemple : ressource A: indispo du 01/02/2024 au 14/02/2024 ensuit 05/03/2024 au 15/03/2024  .. )

*Décrivez les alternatives que vous avez envisagées**
Réserver les créneaux en mode brut // sans que cela ne soit réel. 

**Contexte supplémentaire**
GRR 4.2.2

Merci par avance .

**Comment by ynaessens** [comment]
Bonjour,
dans la version 3.5.1, il existe la possibilité de choisir un type de réservation invisible sur les plannings week_all, month, month_all et year, solution qui a été proposée pour répondre à votre question. Voyez si cela a été porté en 4.2, ou 4.3.
Cordialement,
YN

**Comment by JeromeDevome** [comment]
Bonjour,
Je procède comme cela :
1- Création d'un type de réservation nommé "Indisponible" qui est disponible unique pour les administrateurs.
2- Je créé une réservation avec ce type

L'avantage est que cela peut ce faire par crénaux
